santosfamilyfoundation/RuinaMain
# [RUIN-211] - Add Loading Spinner to Quick Survey

# Description

This fixes a code regression. After the Native Base transition, there was a condition checking if the data was loading was left out. Now the condition has been added with the correct components.

# Images

## Before
![deepin-screen-recorder_Select area_20211117143444](https://user-images.githubusercontent.com/42981026/142270271-6424b90d-8deb-452a-a376-898a3500f82e.gif)

## After
![deepin-screen-recorder_Select area_20211117143000](https://user-images.githubusercontent.com/42981026/142269805-22d0ed01-e24c-4af9-98e2-bf104edab35c.gif)

# Testing

1. Open the app
2. Click "Continue Report"
3. Loading spinner should appear
4. Dialog appears notifying you that the report successfully loaded
5. The form section page should appear
